test/python/sdpa/frost/test_sdpa_fwd_fp8_sm100.py

```
@@ -170,16 +168,15 @@ def _ref_kwargs(sdpa_kwargs):
     return out
 
 
-def _check(O, O_ref, out_dt, in_key, amax_s, amax_s_ref, amax_o, amax_o_ref):
+def _check(O, O_ref, out_dt, in_key, amax_o, amax_o_ref):
```

Ruff (0.16.1)

[error] Ambiguous variable name: `O`

(E741)